assets/javascripts/kitten/components/icons/diamond-icon.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import COLORS from '../../constants/colors-config'
+
+export const DiamondIcon = ({ title, color, ...others }) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="19"
+    height="28"
+    viewBox="0 0 19 28"
+    {...others}
+  >
+    {title && <title>{title}</title>}
+    <path
+      fill={color}
+      fill-rule="evenodd"
```

Comment:
Vu qu'on est sur du jsx, il me semble que tu devrais avoir un warning à cause de la casse de `fill-rule`
```suggestion
      fillRule="evenodd"
```